src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -244,4 +249,20 @@ private function createServerProcess(OutputInterface $output, $address, $documen
 
         return new Process('exec '.$script, $documentRoot, null, null, null);
     }
+
+    private function launchBrowser($output, $address)
+    {
+        $cmdOpen = 'xdg-open';
+        if (false !== stripos(PHP_OS, 'win')) {
```

Comment:
To detect Windows builds I would just check the value of the `DIRECTORY_SEPARATOR` constant as we do everywhere else:

```php
'\\' === DIRECTORY_SEPARATOR
```